source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -24,23 +24,23 @@ When managing these credentials, you must:
 * never use credentials associated with another individual
 * encourage the rest of your team to avoid sharing credentials unnecessarily (technical limitations may mean this is sometimes unavoidable)
 
-## For accounts with organisation-level access
+### Managing accounts with organisation-level access
 
-Often third-party services natively support organisation or team level access, for example the use of alphagov on GitHub. In these cases you should ensure only the appropriate individuals on your team have access.
+Often third-party services natively support organisation or team level access, for example the use of [alphagov on GitHub](https://github.com/alphagov). In these cases you should make sure only the appropriate individuals on your team have access.
 
 With team or organisation-level accounts, where possible you should:
 
-* create separate accounts for each individual rather than share credentials
+* create separate accounts for each individual, rather than share credentials
 * use a single-sign-on provider rather than create an account with a separate username and password - preferably use G Suite as it’s a core part of the GDS leaver process, which means that access to linked third-party services will be revoked automatically when someone leaves
-* ensure that you have a documented process for removing individuals’’s access when they leave GDS
+* make sure that you have a documented process for removing individuals’s access when they leave GDS
 
-##For accounts with individual-level access
+### Managing accounts with individual-level access
 
-When services don’t support organisation-level access, you may have to share individuals’ sign-on credentials.
+When services do not support organisation-level access, you may have to share individuals’ sign-on credentials.
```

Comment:
Our [styleguides prefer contractions](https://www.gov.uk/guidance/content-design/writing-for-gov-uk#contractions), as a general rule.